packages/types/src/codec/Tuple.spec.ts

```diff
@@ -8,9 +8,10 @@ import { VoteThreshold } from '../interfaces/elections';
 import { BlockNumber } from '../interfaces/runtime';
 import { CodecTo } from '../types';
 
+import Metadata from '@polkadot/api-metadata/Metadata';
```

Comment:
I don't believe we should muddle things - we should only have a single class exposed from `api-metadata`  and work with that throughout. So would really suggest `import Metadata from '@polkadot/api-metadata';` where this is the holder object.